src/python/py_generate_pipeline.cpp

```diff
@@ -392,26 +393,10 @@ py::object call_common_generate(
     const utils::PyBindStreamerVariant& py_streamer, 
     const py::kwargs& kwargs
 ) {
-    auto updated_config = update_config_from_kwargs(config, kwargs);
+    auto updated_config = ov::genai::pybind::utils::update_config_from_kwargs(config, kwargs);
```

Comment:
Should `update_config_from_kwargs` be removed from this file as it moved to utils?

Reply:
I removed it before, but accidentally returned it during merging of updated branch.  Thanks, removed it.